v0/destinations/hs/util.js

```diff
@@ -465,8 +467,8 @@ const getExistingData = async (inputs, destination) => {
 
     if (searchResponse.status !== 200) {
       throw new CustomError(
-        "[HS]:: rETL - Error during searching object record.",
-        400
```

Comment:
can we add corresponding test case too ?

Reply:
done